internal/routes/all_routes.go

```diff
@@ -7,16 +7,28 @@ import (
 	"github.com/RHEnVision/provisioning-backend/internal/config"
 	s "github.com/RHEnVision/provisioning-backend/internal/services"
 	"github.com/go-chi/chi/v5"
+	redoc "github.com/go-openapi/runtime/middleware"
 	"github.com/redhatinsights/platform-go-middlewares/identity"
 )
 
+func redocMiddleware(handler http.Handler) http.Handler {
```

Comment:
If lukas is ok with the implementation now, I guess it is fine, but I remember us talking about splitting this into another PR, or am I mistaken? :)